status.go

```diff
@@ -33,13 +33,16 @@ func (e *Executor) isTaskUpToDate(ctx context.Context, t *taskfile.Task) (bool,
 		return false, nil
 	}
```

Comment:
while we are here, there's really no point in this if statement. It's redundant, and only clutters this function.